jenkins-master/jobs/scripts/workspace/runtests.py

```diff
@@ -4,231 +4,137 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import optparse
+import argparse
 import os
+import shutil
 import subprocess
 import sys
-import uuid
 
 here = os.path.dirname(os.path.abspath(__file__))
+venv_path = os.path.join(here, 'tests_venv')
 
 
-class Runner(object):
-
-    def __init__(self, venv_path):
-        self.venv_path = venv_path
-        self.build_path = os.path.join(here, 'build')
-
-        self.s3_bucket = None
-
-        os.chdir(here)
-        self.activate_venv()
-
-    def activate_venv(self):
-        os.chdir(os.path.join(here, 'firefox-ui-tests'))
-        command = ['python', 'create_venv.py', self.venv_path]
+# Create the test environment and activate it
+# TODO remove once we make use of the mozharness script
+import environment
+command = ['python', os.path.join(here, 'firefox-ui-tests', 'create_venv.py'),
```

Comment:
This is only temporarily. With the next PR (switch to mozharness) this code is not necessary anymore because mozharness takes care of the environment creation. Because of that I don't want to pollute `environment.py` with this call.